src/preprocess/preprocess.go

```diff
@@ -556,6 +556,14 @@ func exposeAPIs_main(input string) string {
 			return fmt.Sprintf("Spicetify.Snackbar.enqueueImageSnackbar=%s", submatches[0])
 		})
 
+	// Home Config >= 1.2.45
+	utils.ReplaceOnce(
```

Comment:
put it in the `apply.go`, add another line in the `filesToModified`:
```diff
- filepath.Join(appsFolderPath, "xpui", "home-v2.js"):             insertHomeConfig,
+ filepath.Join(appsFolderPath, "xpui", "home-v2.js"):             insertHomeConfig,
+ filepath.Join(appsFolderPath, "xpui", "xpui.js"):                insertHomeConfig,
```
and then just put it in the if statement on the line 351

Reply:
why would i put it in the if statement? can i just 
```js
func insertHomeConfig(jsPath string, flags Flag) {
	if !flags.HomeConfig {
		return
	}

	utils.ModifyFile(jsPath, func(content string) string {
		utils.ReplaceOnce(
			&content,
			`(createDesktopHomeFeatureActivationShelfEventFactory.*?)([\w\.]+)(\.map)`,
			func(submatches ...string) string {
				return fmt.Sprintf("%sSpicetifyHomeConfig.arrange(%s)%s", submatches[1], submatches[2], submatches[3])
			})

		// >= 1.2.45
		utils.ReplaceOnce(
			&content,
			`(&&"HomeShortsSectionData".*\],)([a-zA-Z])(\}\)\()`,
			func(submatches ...string) string {
				return fmt.Sprintf("%sSpicetifyHomeConfig.arrange(%s)%s", submatches[1], submatches[2], submatches[3])
			})
		return content
	})
}
```